static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -115,11 +118,30 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'A
 			var originalFile = resultObject.getDatastream("original_file");
 			items["viewFile"] = {name : "View File"
 				+ " ("+ StringUtilities.readableFileSize(originalFile['fileSize']) + ")"};
-
-			items["viewFits"] = {name: "View FITS"}
 		}
-		if ($.inArray('viewHidden', metadata.permissions) != -1) {
-			items["viewEventLog"] = {name : "View Event Log"};
+
+		if (viewHidden || isFile || /md_descriptive|tech_md/ig.test(datastreams)) {
+			items["metadata"] = {name: "View metadata", items: {}}
+
+			if (/md_descriptive/ig.test(datastreams)) {
+				items['metadata']['items']["viewMods"] = {name: "View MODS"};
+			}
+
+			if (/md_descriptive_history/ig.test(datastreams)) {
+				items['metadata']['items']["viewModsHistory"] = {name: "View MODS History"};
+			}
+
+			if (isFile) {
```

Comment:
Both FITS and its history require viewHidden. Although I actually don't think there's any way to get to a file object in the admin ui that you don't have a staff role for. See https://github.com/UNC-Libraries/Carolina-Digital-Repository/blob/fcrepo4/fcrepo-clients/src/main/java/edu/unc/lib/dl/model/DatastreamType.java for which permissions datastreams require.